### PR TITLE
Update light.mqtt.markdown

### PR DESCRIPTION
**Pull request by michal-bartak**
Main motivation is to add missing `color_temp` variable to description of `command_on_template`. Then units/values expected by the system.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hello @michalk-k,

When attempting to inspect the commits of your pull request for CLA signature status among all authors we encountered commit(s) which were not linked to a GitHub account, thus not allowing us to determine their status(es).

The commits that are missing a linked GitHub account are the following:

- [`b9bf3f4bab3c64f850014f954317794e534f2a9e`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;home-assistant.io&#x2F;pull&#x2F;20392&#x2F;commits&#x2F;b9bf3f4bab3c64f850014f954317794e534f2a9e) - This commit has something that looks like an email address (michal.bartak@betsys.com). Maybe try linking that to GitHub?.

Unfortunately, **we are unable to accept this pull request until this situation is corrected.**

Here are your options:

1. If you had an email address set for the commit that simply wasn't linked to your GitHub account you can link that email now and it will retroactively apply to your commits. The simplest way to do this is to click the link to one of the above commits and look for a blue question mark in a blue circle in the top left. Hovering over that bubble will show you what email address you used. Clicking on that button will take you to your email address settings on GitHub. Just add the email address on that page and you're all set. GitHub has more information about this option [in their help center](https://help.github.com/articles/setting-your-email-in-git/#commits-on-github-arent-linking-to-my-account).

2. If you didn't use an email address at all, it was an invalid email, or it's one you can't link to your GitHub, you will need to change the authorship information of the commit and your global Git settings so this doesn't happen again going forward. GitHub provides some great instructions on how to change your authorship information [in their help center](https://help.github.com/articles/setting-your-email-in-git/).
   * If you only made a single commit you should be able to run
      ```
      git commit --amend --author="Author Name <email@address.com>"
      ```
      (substituting Author Name and email@address.com for your actual information) to set the authorship information.
   * If you made more than one commit and the commit with the missing authorship information is not the most recent one you have two options:
       1. You can re-create all commits missing authorship information. This is going to be the easiest solution for developers that aren't extremely confident in their Git and command line skills.
       2. You can use [this script](https://help.github.com/articles/changing-author-info/) that GitHub provides to rewrite history. **Please note:** this should be used only if you are very confident in your abilities and understand its impacts.
   * Whichever method you choose, I will come by to re-check the pull request once you push the fixes to this branch.

We apologize for this inconvenience, especially since it usually bites new contributors to Home Assistant. We hope you understand the need for us to protect ourselves and the great community we all have built legally. The best thing to come out of this is that you only need to fix this once and it benefits the entire Home Assistant and GitHub community.

Thanks, I look forward to checking this PR again soon! :heart:

**Comment by homeassistant** [comment]
Hello @michalk-k,

When attempting to inspect the commits of your pull request for CLA signature status among all authors we encountered commit(s) which were not linked to a GitHub account, thus not allowing us to determine their status(es).

The commits that are missing a linked GitHub account are the following:

- [`3c2c790eda6d79d13ba65eaa5c746ee38c708137`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;home-assistant.io&#x2F;pull&#x2F;20392&#x2F;commits&#x2F;3c2c790eda6d79d13ba65eaa5c746ee38c708137) - This commit has something that looks like an email address (michal.bartak@betsys.com). Maybe try linking that to GitHub?.

Unfortunately, **we are unable to accept this pull request until this situation is corrected.**

Here are your options:

1. If you had an email address set for the commit that simply wasn't linked to your GitHub account you can link that email now and it will retroactively apply to your commits. The simplest way to do this is to click the link to one of the above commits and look for a blue question mark in a blue circle in the top left. Hovering over that bubble will show you what email address you used. Clicking on that button will take you to your email address settings on GitHub. Just add the email address on that page and you're all set. GitHub has more information about this option [in their help center](https://help.github.com/articles/setting-your-email-in-git/#commits-on-github-arent-linking-to-my-account).

2. If you didn't use an email address at all, it was an invalid email, or it's one you can't link to your GitHub, you will need to change the authorship information of the commit and your global Git settings so this doesn't happen again going forward. GitHub provides some great instructions on how to change your authorship information [in their help center](https://help.github.com/articles/setting-your-email-in-git/).
   * If you only made a single commit you should be able to run
      ```
      git commit --amend --author="Author Name <email@address.com>"
      ```
      (substituting Author Name and email@address.com for your actual information) to set the authorship information.
   * If you made more than one commit and the commit with the missing authorship information is not the most recent one you have two options:
       1. You can re-create all commits missing authorship information. This is going to be the easiest solution for developers that aren't extremely confident in their Git and command line skills.
       2. You can use [this script](https://help.github.com/articles/changing-author-info/) that GitHub provides to rewrite history. **Please note:** this should be used only if you are very confident in your abilities and understand its impacts.
   * Whichever method you choose, I will come by to re-check the pull request once you push the fixes to this branch.

We apologize for this inconvenience, especially since it usually bites new contributors to Home Assistant. We hope you understand the need for us to protect ourselves and the great community we all have built legally. The best thing to come out of this is that you only need to fix this once and it benefits the entire Home Assistant and GitHub community.

Thanks, I look forward to checking this PR again soon! :heart:

**Comment by homeassistant** [comment]
Hi @michalk-k, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20392).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/light.mqtt.markdown [hunk 1]
```diff
@@ -775,7 +775,12 @@ light:
 The `mqtt` light platform with template schema lets you control a MQTT-enabled light that receive commands on a command topic and optionally sends status update on a state topic.
 It is format-agnostic so you can use any data format you want (i.e., string, JSON), just configure it with templating.
 
-This schema supports on/off, brightness, RGB colors, XY colors, color temperature, transitions, short/long flashing and effects.
+This schema supports on/off, brightness, RGB colors, XY colors, white color temperature, transitions, short/long flashing and effects. Following units and value ranges are passed to/obtained from the system.
+* RGB colors, values from a range: 0 - 255
+* Brightness, value from a range: 0 - 255
+* White color temperature: Mireds.
```
Comment (frenck) [inline source/_integrations/light.mqtt.markdown]: These are better fitting in the configuration section with their respective configuration options.

File: source/_integrations/light.mqtt.markdown [hunk 2]
```diff
@@ -833,15 +838,15 @@ brightness_template:
   required: false
   type: string
 color_temp_template:
-  description: "[Template](/docs/configuration/templating/#processing-incoming-data) to extract color temperature from the state payload value."
+  description: "[Template](/docs/configuration/templating/#processing-incoming-data) to extract white color temperature from the state payload value."
```
Comment (frenck) [inline source/_integrations/light.mqtt.markdown]: Please revert this change, this is not how this term is referred to.

File: source/_integrations/light.mqtt.markdown [hunk 3]
```diff
@@ -1044,6 +1049,47 @@ light:
 
 {% endraw %}
 
+### CCT light (brightnes and temperature)
+
+This example comes from configuration of Shelly RGBW Bulb working in White mode.
```
Comment (frenck) [inline source/_integrations/light.mqtt.markdown]:
```suggestion
This example comes from a configuration of Shelly RGBW Bulb working in White mode. 
```

File: source/_integrations/light.mqtt.markdown [hunk 4]
```diff
@@ -1044,6 +1049,47 @@ light:
 
 {% endraw %}
 
+### CCT light (brightnes and temperature)
+
+This example comes from configuration of Shelly RGBW Bulb working in White mode. 
+Max_mireds and min_mireds set white color temperature boundaries to 3000K - 6500K. Notice the same limits are applied in `command_on_template`, but in Kelvin units this time. It's due to conversion from Mireds to Kelvin which causes exceeding boundary values accepted by the device.
```
Comment (frenck) [inline source/_integrations/light.mqtt.markdown]: - Code-fence the configuration properties so they get highlighted.
- Remove the "white" from the color temperature.
- `Mireds` should not be capitalized.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/light.mqtt.markdown (hunk 2 above)
Comment (michal-bartak) [reply source/_integrations/light.mqtt.markdown]: I admit I changed it intentionally since I'm not aware about temperature of color (in context of the light). This attribute can be hardly used with other light model but white. Imo it is confusing.

Can you explain why you want to stick with 'color temperature' term?

File: source/_integrations/light.mqtt.markdown (hunk 1 above)
Comment (michal-bartak) [reply source/_integrations/light.mqtt.markdown]: I was considering that, but it would require to repeat the same information several times.

IMO the best would be reformat whole document, to begin with system expectations. But I would't touch it for obvious reasons. 

May it stay or you insist to move it to config section, mentioning it everytime it's needed?

Comment (frenck) [reply source/_integrations/light.mqtt.markdown]: > or you insist to move it to config section

Yes.

File: source/_integrations/light.mqtt.markdown (hunk 2 above)
Comment (frenck) [reply source/_integrations/light.mqtt.markdown]: Because it is called color temperature, "white" has nothing to do with that.

File: source/_integrations/light.mqtt.markdown (hunk 4 above)
Comment (michal-bartak) [reply source/_integrations/light.mqtt.markdown]: I did check for Kelvin units. It seems it has to be lowercase too (while Kelvin scale is uppercase). Are you ok with that change?
Also added new line to example description:

> The code also ensures bi-directional conversion of brightness scale between 0-100 (required by the device) and 0-255 (required by Home Assistant).

Is it ok?

File: source/_integrations/light.mqtt.markdown (hunk 2 above)
Comment (michal-bartak) [reply source/_integrations/light.mqtt.markdown]: You are right

**Comment by homeassistant** [comment]
Hi @michalk-k, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20392).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Review by frenck — APPROVED** [review]
Thanks, @michalk-k 👍
